install_test_deps.sh

```diff
@@ -1,6 +1,6 @@
 # Installs xblock-sdk and dependencies needed to run the tests suite.
 # Run this script inside a fresh virtual environment.
-pip install -e git://github.com/edx/xblock-sdk.git@v0.1.3#egg=xblock-sdk==v0.1.3
+pip install -e git://github.com/edx/xblock-sdk.git@master#egg=xblock-sdk==vmaster
```

Comment:
0.1.3 has iteritems and its failing on python3.
[xblock-sdk==v0.1.3](https://github.com/edx/xblock-sdk/blob/0c987356cba5c5a7d47c98654e030ee4c3cb47c2/workbench/runtime.py#L258)

Reply:
Use a hash instead of master I think.  Also you don't need the `==vmaster`